wpimath/src/main/native/cpp/trajectory/Trajectory.cpp

```diff
@@ -124,6 +124,16 @@ Trajectory Trajectory::RelativeTo(const Pose2d& pose) {
   return Trajectory(newStates);
 }
 
+Trajectory Trajectory::operator+(const Trajectory& other) const {
+  auto states = m_states;
+  auto otherStates = other.States();
+  std::for_each(otherStates.begin(), otherStates.end(),
+                [&](auto& s) { s.t += m_totalTime; });
```

Comment:
Range-based for is less typing:
```suggestion
  for (auto& otherState : otherStates) {
    otherState.t += m_totalTime;
  }
```